demos/flask/app.conf

```diff
@@ -0,0 +1,3 @@
+[app]
+
+skey = e806cf5156fd85b2f384a1181d12e5c9e833f15d
```

Comment:
This is specific for Flask to handle user sessions. If it is not present the callback from the Duo iframe will fail as the user cannot be verified as logged in. It can be scrubbed out, but it is required to have the app run.